neurons/miner.py

```diff
@@ -147,6 +147,11 @@ def get_config():
         default=constants.SUBNET_UID,
         help="The subnet UID.",
     )
+    parser.add_argument(
+        "--use_hotkey_in_hash",
+        action="store_false",  # Defaults to True.
```

Comment:
We should probably default to False for now, until enough valis have updated. Otherwise, the miner will upload with the hotkey and most the valis will think it's a bad model

Reply:
Done.